src/config.cc

```diff
@@ -194,6 +194,7 @@ namespace Astroid {
     default_config.put ("editor.cmd", "gvim -f -c 'set ft=mail' '+set fileencoding=utf-8' '+set enc=utf-8' '+set ff=unix' '+set fo+=w' %1");
     default_config.put ("editor.external_editor", true); // should be true on Wayland
 # endif
+    default_config.put ("editor.external_tmpfile_suffix", "@msg.draft"); // suffix of tmp files passed to external editors; default: '@msg.draft'
```

Comment:
Do we really need this? Is this for editors to automatically set filetype? E.g. the gvim line above sets ft=mail by force anyway. Does this really add any security; couldn't an app easily just use `lsof` to figure out that this is an email file anyway. The same with a draft email which is stored in a users maildir.

Reply:
Two reasons:
1. It echoes the previous behaviour where the message id was used as the temp file name.
2. It helps `emacslient` (which I use ;-) to automatically set filetype

Reply:
Does the tmpfile change name if you change from address? What if you change the from address from within the editor with the file open? I though the point was to hide that this was a e-mail message, with this it will be just as easy to detect.

Reply:
The suffix should probably be `.eml` in that case. Also; perhaps the best would be to completely decouple the file-name from message-id.